src/routes/item/[id]/+page.svelte

```diff
@@ -6,13 +6,9 @@
 	/** @type {import('./$types').PageData} */
 	export let data;
 
-	$: postLink = `/${data.origin}#${data.id}`;
-	$: postCommentLink = `/item/${data.parentId}?origin=${data.origin}#${data.id}`;
+	$: postCommentLink = `/item/${data.parentId}#${data.id}`;
 
-	$: backLink =
-		data.type === 'comment'
-			? data.parentId && postCommentLink
-			: data.origin && postLink;
+	$: backLink = data.type === 'comment' && data.parentId && postCommentLink;
```

Comment:
Can we rename `parentId` to `rootId` or similar to clarify that this always links to the original post?